app/controllers/patients_controller.rb

```diff
@@ -500,6 +500,27 @@ def household_removeable
     render json: { removeable: !duplicate_contact }
   end
 
+  # Check to see if a phone number has blocked SMS communications with SaraAlert
+  def sms_eligibility_check
+    redirect_to(root_url) && return unless current_user.can_edit_patient?
+
+    phone_number = params.permit(:phone_number)[:phone_number]
+    blocked = BlockedNumber.exists?(phone_number: phone_number)
+    render json: { sms_eligible: !blocked }
+  end
+
+  def send_reminder
```

Comment:
I had removed this since it's not something we support anymore (or want to) in the UI. Might have come back during rebase. Make sure to remove this method.